### PR TITLE
checks if an entry point exists during manifest validation

### LINKED ISSUE #925: Check that entry point exists during forc `Manifest` validation
During validation of the `Manifest`, we should check that the specified entry point file actually exists.

### PR DESCRIPTION
**Pull request by eureka-cpu**
Closes #925 

This updates the `validate` method to check that the `path` from the manifest `entry` field exists.

### CODE REVIEW

File: forc-pkg/src/manifest.rs [hunk 1]
```diff
@@ -110,7 +110,13 @@ impl Manifest {
     /// Produces the string of the entry point file.
     pub fn entry_string(&self, manifest_dir: &Path) -> anyhow::Result<Arc<str>> {
         let entry_path = self.entry_path(manifest_dir);
-        let entry_string = std::fs::read_to_string(&entry_path)?;
+        let entry_string = std::fs::read_to_string(&entry_path).map_err(|e| {
+            anyhow!(
+                "failed to validate path from entry point {:?}: {}",
```
Comment (adlerjohn) [inline forc-pkg/src/manifest.rs:115]: Is there a better way of wording this?

Comment (eureka-cpu) [reply forc-pkg/src/manifest.rs:115]: Do we not want to specify that it's attempting to validate a path exists from the entry field?

Comment (adlerjohn) [reply forc-pkg/src/manifest.rs:115]: Ah good point. In that case make it more wordy, to describe that it's checking the entry point field in the manifest file?

**Review by adlerjohn — COMMENTED** [review]
lgtm, but will defer

**Review by JoshuaBatty — APPROVED** [review]

**Pull request merged**